src/Parser/Core/Modules/Haste.js

```diff
@@ -41,7 +41,6 @@ class Haste extends Analyzer {
     [209166]: 0.3, // DEBUFF - Fast Time from Elisande
     [209165]: -0.3, // DEBUFF - Slow Time from Elisande
     // [208944]: -Infinity, // DEBUFF - Time Stop from Elisande
-    [SPELLS.BONE_SHIELD.id]: 0.1, // Blood BK haste buff from maintaining boneshield
```

Comment:
Why did you remove this?

Reply:
It was repeated. I don't have line numbers on my phone but it's right below Icy Veins